src/stores/TextFormStore.ts

```diff
@@ -0,0 +1,12 @@
+import { proxy } from 'valtio'
+
+export default proxy<{
+  text: string
+  waitBurner: boolean
+  loading: boolean
+  error?: unknown
+}>({
+  text: '',
```

Comment:
I'm not sure that we need a store for the TextArea. If you are sure, then I'm okay